manim/scene/scene_file_writer.py

```diff
@@ -77,46 +77,48 @@ def init_output_directories(self, scene_name):
-                        os.path.join(
-                            file_writer_config["video_dir"],
-                            module_directory,
-                            self.get_resolution_directory(),
-                        )
-                    )
-                else:
-                    movie_dir = guarantee_existence(
-                        os.path.join(file_writer_config["video_dir"])
-                    )
-            self.movie_file_path = os.path.join(
-                movie_dir,
-                add_extension_if_not_present(
-                    default_name, file_writer_config["movie_file_extension"]
-                ),
-            )
-            self.gif_file_path = os.path.join(
-                movie_dir,
-                add_extension_if_not_present(default_name, GIF_FILE_EXTENSION),
-            )
+            pass
+        if file_writer_config["video_dir"]:
             if not file_writer_config["custom_folders"]:
-                self.partial_movie_directory = guarantee_existence(
+                movie_dir = guarantee_existence(
                     os.path.join(
-                        movie_dir,
-                        "partial_movie_files",
-                        default_name,
+                        file_writer_config["video_dir"],
+                        module_directory,
+                        self.get_resolution_directory(),
                     )
                 )
             else:
-                self.partial_movie_directory = guarantee_existence(
-                    os.path.join(
-                        file_writer_config["media_dir"],
-                        "temp_files",
-                        "partial_movie_files",
-                        default_name,
-                    )
+                movie_dir = guarantee_existence(
+                    os.path.join(file_writer_config["video_dir"])
                 )
+        self.movie_file_path = os.path.join(
+            movie_dir,
+            add_extension_if_not_present(
+                default_name, file_writer_config["movie_file_extension"]
+            ),
+        )
+        self.gif_file_path = os.path.join(
+            movie_dir,
+            add_extension_if_not_present(default_name, GIF_FILE_EXTENSION),
+        )
+        # The partial movie directory has to be set even if we do not write to movie
+        if not file_writer_config["custom_folders"]:
+            self.partial_movie_directory = guarantee_existence(
+                os.path.join(
+                    movie_dir,
+                    "partial_movie_files",
+                    default_name,
+                )
+            )
+        else:
+            self.partial_movie_directory = guarantee_existence(
+                os.path.join(
+                    file_writer_config["media_dir"],
+                    "temp_files",
+                    "partial_movie_files",
+                    default_name,
+                )
+            )
```

Comment:
Even if this looks like a lot of changes, it is only an identation shift: in the first place, the partial_movie_directory was created only if write_to_movie was set. But there is some cases where you need the directory without the variable.
The overall goal was to remove ffmpeg complaining when building docs that the partial_movie_files file was empty.